### PR TITLE
Removes SE-EDU branding

### PR DESCRIPTION
**Pull request by lightz96**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ ifdef::env-github[]
 :note-caption: :information_source:
 :warning-caption: :warning:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3/tree/master
+:repoURL: https://github.com/AY1920S1-CS2103T-W12-2/main
 
 By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
```
Comment (krusagiz) [inline docs/DeveloperGuide.adoc]: Please update `By:` and `Since:` to remove SEEDU branding.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -12,7 +12,7 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level3
+:repoURL: https://github.com/AY1920S1-CS2103T-W12-2/main
 
 By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
```
Comment (krusagiz) [inline docs/UserGuide.adoc]: Please update `By:` and `Since:` to remove SEEDU branding.

**Review by krusagiz — CHANGES_REQUESTED** [review]
We need to think of a team name and use it consistently throughout the code.

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (lightz96) [reply docs/DeveloperGuide.adoc]: Ok, I have updated.

File: docs/UserGuide.adoc (hunk 2 above)
Comment (lightz96) [reply docs/UserGuide.adoc]: Ok, I have updated

File: README.adoc [hunk 3]
```diff
@@ -1,7 +1,7 @@
 = Address Book (Level 3)
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
-https://travis-ci.org/se-edu/addressbook-level3[image:https://travis-ci.org/se-edu/addressbook-level3.svg?branch=master[Build Status]]
+https://travis-ci.org/AY1920S1-CS2103T-W12-2/main[image:https://travis-ci.org/AY1920S1-CS2103T-W12-2/main.svg?branch=master[Build Status]]
 https://ci.appveyor.com/project/damithc/addressbook-level3[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
```
Comment (limerencee) [inline README.adoc]: We should change the other 3rd party apps to point to our project's context.

**Review by limerencee — CHANGES_REQUESTED** [review]
We should change the context from SE-EDU to THRIFT for the 3rd party apps.

**Comment by limerencee** [comment]
We should change the `AboutUs.adoc` and `ContactUs.adoc` files for this PR as well.

Checklist:
- [ ] `docs\AboutUs.adoc`
- [ ] `docs\ContactUs.adoc`
- [ ] `docs\DeveloperGuide.adoc`
- [ ] `README.adoc`
- [ ] `docs\UserGuide.adoc`

**Review by limerencee — APPROVED** [review]

**Review by krusagiz — APPROVED** [review]

**Pull request merged**